Tools/TestWebKitAPI/Tests/WebKitGLib/TestBackForwardList.cpp

```diff
@@ -73,7 +73,6 @@ class BackForwardListTest: public WebViewTest {
     {
         g_assert_nonnull(item);
         g_assert_cmpstr(webkit_back_forward_list_item_get_uri(item), ==, uri);
-        g_assert_cmpstr(webkit_back_forward_list_item_get_title(item), == , "");
```

Comment:
Even if the function is deprecated, we still want to test it. Instead, what we do is disabling deprecation warnings for tests. For example you can see `G_GNUC_{BEGIN,END}_IGNORE_DEPREFCATIONS` used in a number of API test source files, e.g. `Tests/WebKitGtk/TestContextMenu.cpp`.

Instead, I would suggest this:

```cpp
G_GNUC_BEGIN_IGNORE_DEPRECATIONS;
g_assert_cmpstr(webkit_back_forward_list_item_get_title(item), ==, "");
G_GNUC_END_IGNORE_DEPRECATIONS;
```

Reply:
I agree we should still test deprecated API, but in this case the functionality was removed, so we can just probably remove the test too.